### PR TITLE
Various changes to get the new production host up and running

### PR DESCRIPTION
**Pull request by tlvu**
## Changes

**Non-breaking changes**
- Bootstrap testsuite: only crawl the subset enough to pass canarie-api monitoring: faster when system under test has too much other stuff.
- New script: `check-autodeploy-repos`: to ensure autodeploy will trigger normally.
- New script: `sync-data`: to pull data from existing production host to a new production host or to a staging host to emulate the production host.
- thredds, geoserver, generic_bird: set more appropriate production values, taken from https://github.com/Ouranosinc/birdhouse-deploy/commit/316439e310e915e0a4ef35d25744cab76722fa99
- monitoring: fix redundant `network_mode: host` and `ports` binding since host network_mode will already automatically perform port bindings

**Breaking changes**
- None

## Related Issue / Discussion

- https://github.com/bird-house/birdhouse-deploy-ouranos/pull/16
- https://github.com/Ouranosinc/pavics-vdb/pull/48
- https://github.com/Ouranosinc/ouranos-ansible/pull/2

Fixes https://github.com/bird-house/birdhouse-deploy/issues/212

@tlogan2000 @moulab88 FYI

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -14,7 +14,26 @@
 [Unreleased](https://github.com/bird-house/birdhouse-deploy/tree/master) (latest)
 ------------------------------------------------------------------------------------------------------------------
 
-[//]: # (list changes here, using '-' for each new entry, remove this when items are added)
+## Changes
+
+- Various changes to get the new production host up and running
+
+    **Non-breaking changes**
+    - Boostrap testsuite: only crawl the subset enough to pass canarie-api monitoring: faster when system under test has too much other stuff.
```
Comment (matprov) [inline CHANGES.md:22]: `Boostrap` in here seems more to be a kind of music than a stack deployment concept ;)

File: birdhouse/vagrant-utils/install-docker.sh [hunk 2]
```diff
@@ -97,5 +97,6 @@ LATEST_COMPOSE_VERSION="`git ls-remote https://github.com/docker/compose | grep
 # LATEST_COMPOSE_VERSION=$(curl --silent https://api.github.com/repos/docker/compose/releases/latest | jq .name -r)  # need jq :(
 sudo curl -L "https://github.com/docker/compose/releases/download/$LATEST_COMPOSE_VERSION/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
 sudo chmod +x /usr/local/bin/docker-compose
-sudo curl -L "https://raw.githubusercontent.com/docker/compose/${LATEST_COMPOSE_VERSION}/contrib/completion/bash/docker-compose" -o /etc/bash_completion.d/docker-compose
+# docker-compose v2 removed the completion file, see https://github.com/docker/compose/issues/8550
+#sudo curl -L "https://raw.githubusercontent.com/docker/compose/${LATEST_COMPOSE_VERSION}/contrib/completion/bash/docker-compose" -o /etc/bash_completion.d/docker-compose
```
Comment (matprov) [inline birdhouse/vagrant-utils/install-docker.sh:101]: Is there a value in keeping this comment here?

**Review by matprov — APPROVED** [review]

File: birdhouse/vagrant-utils/install-docker.sh (hunk 2 above)
Comment (tlvu) [reply birdhouse/vagrant-utils/install-docker.sh:101]: I was half-half on this one.  That line will work for docker-compose v1.  Given issue https://github.com/bird-house/birdhouse-deploy/issues/246, we could potentially use v1 for a bit longer.

File: CHANGES.md (hunk 1 above)
Comment (tlvu) [reply CHANGES.md:22]: I meant "bootstrap" LOL, typo :D

**Pull request merged**